browser/src/core/SocketWorker.js

```diff
@@ -0,0 +1,112 @@
+/* -*- js-indent-level: 8; fill-column: 100 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/* eslint-disable no-inner-declarations */
+/* eslint no-unused-vars: ["warn", { "argsIgnorePattern": "^_" }] */
+/* global importScripts Uint8Array cool */
+
+if ('undefined' === typeof window) {
+	importScripts('../../node_modules/fzstd/umd/index.js');
+	importScripts('../layer/tile/CanvasTileUtils.js');
```

Comment:
I suspect that this has not been tested with a bundlified product build (?) or do we distribute CanvasTileUtils.js on its own ?